src/installation/index.md

```diff
@@ -1,117 +1,129 @@
 
-To install `espup`, use the following command:
-```sh
-cargo install espup
-```
-You can also directly download pre-compiled [release binaries] or use [cargo-binstall].
+3. To build `std` projects, you also need to install:
+    - [`LLVM`][llvm-website] compiler infrastructure
+    - Other [`std` developemnt requirements][rust-esp-book-std-requirements]
+    - In your project's file `.cargo/config.toml`, add the unstable Cargo [feature][cargo-book-unstable-features] `-Z build-std`. Our [template projects][rust-esp-book-write-app-generate-project] that are discussed later in this book already have it added.
 
-Once that `espup` is installed, you can simply run:
-```sh
-espup install
-```
 
-This will install all the necessary tools to develop Rust applications for all supported ESP targets.
+[llvm-website]: https://llvm.org/
+[rust-esp-book-std-requirements]: #std-development-requirements
+[cargo-book-unstable-features]: https://doc.rust-lang.org/cargo/reference/unstable.html
+[rust-esp-book-write-app-generate-project]: ../writing-your-own-application/generate-project-from-template.md
+
+
+Now you should be able to build and run projects on the Espressif's `RISC-V` chips.
+
+
+## RISC-V and Xtensa targets
+
+[`espup`][espup-github] is a tool that simplifies installing and maintaining the components required to develop Rust applications for the `Xtensa` architecture.
 
-`espup` will create an export file, by default `$HOME/export-esp.sh` on Unix systems and `%USERPROFILE%\export-esp.ps1` on Windows. This file contains some environment variables required to build projects. Make sure to source this file in every terminal before building any application:
+[espup-github]: https://github.com/esp-rs/espup
 
+To install `espup`, run:
 ```sh
-# Unix
-. $HOME/export-esp.sh
-# Windows
-%USERPROFILE%\export-esp.ps1
+cargo install espup
 ```
 
-[esp-rs/espup]: https://github.com/esp-rs/espup
-[see Usage section of the `espup` Readme]: https://github.com/esp-rs/espup#usage
+You can also directly download pre-compiled [release binaries] or use [cargo-binstall].
+
 [release binaries]: https://github.com/esp-rs/espup/releases
 [cargo-binstall]: https://github.com/cargo-bins/cargo-binstall
 
+Once `espup` is installed, do the following:
+
+1. Install all the necessary tools to develop Rust applications for all supported Espressif targets by running:
+    ```sh
+    espup install
+    ```
+
+    `espup` will create an export file that contains some environment variables required to build projects:
+
+    - Unix systems - `$HOME/export-esp.sh`
+    - Windows - `%USERPROFILE%\export-esp.ps1`
+
+2. On Unix systems, make sure to source this file in every terminal before building any application: `. $HOME/export-esp.sh`
+
+  > On Windows systems, the file is created to show the user the modified enviroment variables, but user does not require sourcing any file.
```

Comment:
@SergioGasquez 

I think the fact that no actions are required is more important to the user than the reason behind it. Do you think we can rewrite the paragraph like this:

On Windows systems, no need to source the file. It is only created to show the modified environment variables.